### PR TITLE
[* DateTimeV2] Fixed misinterpretation of a relative duration if prefixed by a number (#2632)

### PR DESCRIPTION
**Pull request by LionbridgeCSII**
Fix to issue #2632.

Test cases added to English DateTimeModel.

### CODE REVIEW

File: Specs/DateTime/English/DateTimeModel.json [hunk 1]
```diff
@@ -20627,5 +20627,80 @@
...
+              "start": "2016-10-01",
+              "end": "2016-11-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "We will be only 3 next week.",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20657]: Can you add some cases with the plurals too, if they don't exists? Like "We'll be gone for only the 3 next weeks".
Please add the corrected support specs in a couple other languages too.

**Review by tellarin — CHANGES_REQUESTED** [review]
Looks good. Just some more specs pending. Thanks!

File: Specs/DateTime/English/DateTimeModel.json (hunk 1 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json:20657]: added

Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json:20657]: also in French, Portuguese and Spanish

**Review by tellarin — APPROVED** [review]

**Pull request merged**